### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Athaliah8**
##In This repo
[ ]Added media queries 
[ ]Added addition html elements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,14 +11,34 @@
...
+              <label for="checkbox_toggle" class="hamburger">&#9776;</label>
+              <!-- NAVIGATION MENUS -->
+              <div class="menu">
+                <li><a href="/">Portfolio</a></li>
+                <li><a href="/">About</a></li>
+                <li><a href="/">Contact</a></li>
+             
+        
                
           </nav>
```
Comment (andgarzonmal) [inline index.html:30]: - [ ]  Kyndly stick to the figma template as much as you can,  please notice that in the figma template the navbar does not take the 100% of the width of the page, there are some spaces on the right and left side of the navbar. ✏️

Your design 

image

Figma template

image

File: index.html [hunk 2]
```diff
@@ -116,7 +159,7 @@ <h1>Tonic</h1>
                  <p class="project_description">
                   A daily selection of privately personalized reads; no accounts or sign-ups required.
                  </p>
-            </div>
+
```
Comment (andgarzonmal) [inline index.html:162]: - [ ]  Kyndly stick to the figma design as much as posible, in the cards section you added too much margin to the left and rigt side of the cards in comparison to the figma design.

## Figma design 

image

## Your design

image

File: index.html [hunk 3]
```diff
@@ -125,14 +168,20 @@ <h1>Tonic</h1>
                </ul>
                <div class="external_link">
                    <a href="#">See Project</a>
-               </div>
+
```
Comment (andgarzonmal) [inline index.html:171]: - [ ]  Kyndly stick to the figma design as much as posible, in the cards section you have to place the card images according to the figma design, please notice that in some cards the image is on the right side and in others the image is on the left side.

## Figma design
image

## Your design 

image

File: index.html [hunk 4]
```diff
@@ -155,7 +204,8 @@ <h1>Multi-Post
                </ul>
                <div class="external_link">
                    <a href="#">See Project</a>
-               </div>
+               
+              </div>
```
Comment (andgarzonmal) [inline index.html:208]: - [ ]  Kyndly stick to the figma design as much as posible please notice you have several diferences in the About myself section .

## Your design 
image

## Figma design 

image

File: index.html [hunk 5]
```diff
@@ -238,6 +288,7 @@ <h2 class="first-text-contact">Contact me</h2>
 
 
             </section>
-   
+
```
Comment (andgarzonmal) [inline index.html:291]: Kyndly stick to the figma design as much as posible, please notice in the contact form the form does not take to much with of the page.

## Figma design 
image

## Your design
image

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @Athaliah8    ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 6]
```diff
@@ -4,17 +4,121 @@
...
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+}
```
Comment (Gopxfs) [inline style.css:43]: As requested by the previous reviewer, kindly make sure that your `nav` element doesn't occupy the whole width 😄:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

Check the space between the elements and the body border in the original design 😉.

> Note: you can always click on the images to see them with more details.

File: index.html [hunk 7]
```diff
@@ -96,14 +133,21 @@ <h1>Multi-Post
...
               </footer>
   
             
           </section>
           <section id="p3">
             <div id="introd_3">
+              
+                <div class="row">
+                  <div class="column">
+
```
Comment (Gopxfs) [inline index.html:147]: As requested by the previous reviewer, kindly update the margin on your projects section. Currently, there is too much space between the card and the body border 😉.

| Figma design  | Your design |
| ------------- | ------------- |
|   |   |

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.

File: index.html [hunk 8]
```diff
@@ -96,14 +133,21 @@ <h1>Multi-Post
...
+              
+                <div class="row">
+                  <div class="column">
+      
                   <img  class="project_image" src="./images/Snapshoot Portfolio (2).svg" alt="snapshoot image project" border-raduis:8px>
+                  </div>
+                  <div class="column" >
                   <h1>Tonic</h1>
                   <div id="general_inf_2">
                       <p class="title">CANOPY</p>
```
Comment (Gopxfs) [inline index.html:153]: As requested by the previous reviewer, kindly make sure to make your project's images intersect between left and right, as displayed below 😉:

| Figma design  | Your design |
| ------------- | ------------- |
|   |   |

File: style.css [hunk 9]
```diff
@@ -401,10 +473,12 @@ footer h3 {
...
 #about_me {
-  height: 130vh;
-  margin: 0 0 0 16px;
+  height: 180vh;
+  margin: 3px;
   display: flex;
   flex-direction: column;
+  background-color: #fff;
+  border-radius: 0 10% 0 0;
 }
```
Comment (Gopxfs) [inline style.css:482]: As requested by the previous reviewer, please make sure to apply the desktop design displayed on Figma 😉. Please note that, currently, you still have your mobile version styling:

| Figma design  | Your design |
| ------------- | ------------- |
|   |   |

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @Athaliah8 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Implemented some of the changes requested by the previous reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**